db/schema.rb

```diff
@@ -91,6 +91,17 @@
     t.index ["user_id"], name: "index_journeys_on_user_id", using: :btree
   end
 
+  create_table "logs", force: :cascade do |t|
+    t.integer  "minutes"
```

Comment:
* is `logs` a good model/table name? I cannot imagine what a Log is

* Use `null: false` (probably)